### PR TITLE
Iss3 search

### PR DESCRIPTION
**Pull request by Grinsburg**

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -1,12 +1,11 @@
-body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+html {
+  position: relative;
```
Comment (yamixail) [inline src/index.css]: This property is redundant as every `position: absolute` element is positioned relative to document by default

File: src/index.css [hunk 2]
```diff
@@ -1,12 +1,11 @@
...
+  min-height: 100%;
 }
 
+body{
+  padding-bottom: 70px;
```
Comment (yamixail) [inline src/index.css]: Please use containers for this padding

Comment (Grinsburg) [reply src/index.css]: oh, yes

File: src/index.css (hunk 1 above)
Comment (Grinsburg) [reply src/index.css]: change relative to absolute?

Comment (yamixail) [reply src/index.css]: You don't need this property at all. What was the reason of using it?

Comment (Grinsburg) [reply src/index.css]: i removed this property

File: src/index.css [hunk 3]
```diff
@@ -1,10 +1,5 @@
-body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+html {
+  min-height: 100%;
```
Comment (yamixail) [inline src/index.css]: Does the `html` collapsed to less then 100%?

**Pull request merged**